backend/database/migrations/2022_08_11_112450_create_companies_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Internships\Enums\CompanyStatus;
+use Internships\Models\User;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::create("companies", function (Blueprint $table): void {
+            $table->id();
+            $table->string("name");
+            $table->text("description");
+            $table->foreignIdFor(User::class)->constrained()->cascadeOnDelete();
+            $table->json("address");
+            $table->string("logo")->nullable();
```

Comment:
`logo` will be path to the file or file content encoded as base64? Maybe better use `blolb` (binary) field?
https://laravel.com/docs/9.x/migrations#column-method-binary

Reply:
> `logo` will be path to the file or file content encoded as base64? Maybe better use `blolb` (binary) field? https://laravel.com/docs/9.x/migrations#column-method-binary

Support for logos will be implemented in a separate pull request. Then, most likely, the field type will also be changed.